lang/en.yml

```diff
@@ -0,0 +1,24 @@
+en:
+  SHAREDRAFT:
```

Comment:
There keys don't seem to match the normal naming convention of a fully namespaced class and mixed case keys, here everything is just all caps.  Is that a standard convention for template only keys or does something need to change here?

Reply:
All caps vs mixed case is definitely all over the place (check out https://github.com/silverstripe/silverstripe-admin/blob/1/lang/en.yml) - I _think_ there's more all caps than not? But Not sure.

Agree about class name though. These templates are used by `ShareDraftController` specifically so I'll attach them to that.